packages/dev/biome-config/configs/biome-base.json

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
🟡 If possible, let's use `.jsonc` [file extension](https://biomejs.dev/internals/language-support/#jsonc-support). It should allow us to drop any comments if necessary. It can be handy in case we need to explain settings in this config.

Reply:
done